client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.4.0
     api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"
```

Comment:
this also has to be updated in all the files

Reply:
You meant I should update the api-approved to the current PR 419?
`api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/419"`?

Sure.  That makes sense.

Reply:
Done